keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -841,8 +843,15 @@ in back-to-back releases.
 -->
 
 #### Alpha
+
 - Feature implemented behind a feature flag
-- The strategy `MinimumCompatibilityVersionStrategy` is implemented
+- The strategy `OldestEmulationVersion` is implemented
+
+#### Beta
+
+- e2e tests
```

Comment:
which do we know we would need?

Reply:
Added example list. Feel free to expand if its missing cases